pandas/core/arrays/integer.py

```diff
@@ -652,7 +652,7 @@ def _reduce(self, name, skipna=True, **kwargs):
             data[mask] = self._na_value
 
         op = getattr(nanops, "nan" + name)
-        result = op(data, axis=0, skipna=skipna, mask=mask)
+        result = op(data, axis=0, skipna=skipna, mask=mask, **kwargs)
```

Comment:
which args are meant to be passed in?

Reply:
This is supposed to let the ddof argument to be passed in here in order to fix the issue.